shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/algorithm/sharding/cosid/PropertiesUtilTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sharding.algorithm.sharding.cosid;
+
+import org.junit.Test;
+
+import java.util.Properties;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class PropertiesUtilTest {
+    
+    @Test
+    public void assertGetRequiredValue() {
+        Properties properties = new Properties();
+        properties.setProperty("sql-show", "true");
+        String targetValue = PropertiesUtil.getRequiredValue(properties, "sql-show");
```

Comment:
It may be better to change `targetValue` to `actual`.

Reply:
@yx9o Received, the modification has been completed.

Reply:
@acmenlt Sorry, I meant to change to `String actual = `, what do you think?

Reply:
@yx9o I looked through other codes and found that there is such a writing method. I'll correct it again.